app/src/main/java/io/github/androidacademyglobal/profile/Previews.kt

```diff
@@ -0,0 +1,62 @@
+package io.github.androidacademyglobal.profile
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.tooling.preview.Preview
+import io.github.androidacademyglobal.R
+import io.github.androidacademyglobal.ui.theme.AndroidAcademyAppTheme
+
```

Comment:
Maybe there is a point in giving more specific names to files, e.g. `ProfilePreviews.kt`?
So that it would be easier to navigate across the project

Reply:
good point, renamed it